editor/editor_layouts_dialog.cpp

```diff
@@ -0,0 +1,108 @@
+#include "editor_layouts_dialog.h"
```

Comment:
Could you add the Godot copyright header? See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes